examples/httpserver/bin/hello_world_server_secure.dart

```diff
@@ -28,8 +28,7 @@ Future main() async {
   );
   print('Listening on localhost:${server.port}');
   await for (HttpRequest request in server) {
-    request.response
-      ..write('Hello, world!')
-      ..close();
+    request.response.write('Hello, world!');
+    await request.response.close();
```

Comment:
Same question: see https://github.com/dart-lang/dart-tutorials-samples/blob/master/httpserver/bin/hello_world_server_secure.dart

Reply:
Aye, I'll do a full sweep of the tutorial samples

Reply:
Also somewhere in https://github.com/dart-lang/dart-tutorials-samples/pull/49